docs/usage/secrets_rotation.md

```diff
@@ -18,3 +23,43 @@ To put it in use, a cloudprovider secret is bound to one more namespaces (and th
 1. Update the data keys in the secret.
 2. :warning: Wait until all Shoots using the secret are reconciled before you disable the old secret in your infrastructure account! Otherwise the shoots will no longer function.
 3. After all Shoots using the secret were reconciled you can go ahead and deactivate the old secret in your infrastructure account.
+
+### Gardener generated secrets
+
+####Kubeconfig
+
+*Name*: <shoot-name>.kubeconfig
+
+*Description*: Admin Kubeconfig provided by Gardener for the managed shoot cluster.
+
+---
+**NOTE**
+
+This Kubeconfig contains the highest privileges in the cluster. We strongly discourage distributing or using this Kubeconfig
+on a daily basis. Use dedicated [Service Accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/),
+[OIDC](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens) or similar alternatives
+to grant role-based and revocable access for a broader audience.
+
+---
+
+*Rotation*: Kubeconfig can be rotated by annotating the shoot resource with `gardener.cloud/operation: rotate-kubeconfig-credentials`.
```

Comment:
Long time ago this kubeconfig was containing a client certificate that cannot be revoked. See https://github.com/gardener/gardener/pull/1339 for more details. Should we mention this limitation in the documentation?